### ISSUE TITLE
Start with no ROM

### ISSUE THREAD

**Issue by inactive123** [Themaister/RetroArch-Phoenix#12]
Certain emulators/games engines (such as ScummVM and VICE) require booting with no input ROM at all. 

With the ROM filtering added recently, there is no way anymore to launch RetroArch with a core and a 'dummy file'. Instead, we should add a button that will allow us to boot a core without any ROMs fed.

Also - there should be a 'boot in RGUI' option as well - this would launch RetroArch with RGUI at startup just like the console ports.

**Comment by Themaister** [comment]
Why would they require booting with no input ROM? They implement their own file browsing or something, and it's impossible to load something immediately?

If they require a dummy rom, valid_extensions should be NULL already, and any rom will be allowed in Phoenix at least.

**Comment by inactive123** [comment]
Yes, SCUMMVM has a built-in menu that requires you to select a particular dir and based on that it will detect which game it's going to add. AFAIK there is no commandline option to 'feed it' a game, or if there is, it's not commonly used.

You can compile from it here -

https://github.com/libretro/scummvm

go to backends/platform/libretro/build and type make.
